corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js

```diff
@@ -415,6 +418,11 @@ hqDefine('app_manager/js/detail-screen-config.js', function () {
             self.add_item('responses');
         }
 
+        self.lookup_attach_results = ko.observable(state.lookup_display_results);  // We don't store lookup_attach_results
```

Comment:
@orangejenny, @phillipm, I'm only using the "Attach results to case list" checkbox to determine whether to show the rest of the form. This feels wrong, but I couldn't see in the spec anything else that this value is used for. Did I miss something? If not, could we drop the checkbox, and just use the "Display results in case list" checkbox to determine whether to show/hide the next two input fields?